### PR TITLE
fix(boinc): fix kaktwoos validator regex and experimental perf increase

### PR DESCRIPTION
**Pull request by Hyenadae**
With @BalintCsala and his changes, we get roughly 15-20% more performance in our OpenCL kernel, but some GPUs benefit this more than others...
**Edit: Looking to split kernels up, one for AMD and one for Nvidia!**

"kaktwoos-amd.cl"
Vega 56 went from 42 -> 63m/s (million seeds a second) (+50%!)
RX 480 went from 17 -> 25m/s (+30%!)
r5 540M went from 3.9 -> 5.35m/s (+30%!)

"kaktwoos-nv.cl"
RTX 3090 went from 225 -> 232m/s (+5%?)
RTX 2060 went from 52 -> 80m/s  (+60%)
GTX 1650M went from 50 -> 52m/s (+5%)
GTX 1080ti went from 61 -> 58.4m/s (-3%) (might be due to more kernel printing, or heating?)
GTX 1070 had *no change* in perf, 53.3m/s

Please verify seeds and outputs further
All changes will be boinc version.xml side and in kaktwoos repo. No binary updates required

### CODE REVIEW

**Comment by BoySanic** [comment]
Do we want a version number increment or is it not necessary?

**Comment by Hyenadae** [comment]
It's kinda necessary as it helps BOINC push stuff through in an obvious way

**Comment by BoySanic** [comment]
Nevermind I misunderstood what I was reading. 

I'm just gonna run our baseline WU to make sure I get the same result and then otherwise it looks good to me

**Review by hube12 — CHANGES_REQUESTED** [review]

**Comment by Hyenadae** [comment]
Alright, so we're making progress towards testing enough GPUs to confirm that the 'AMD' and 'Nvidia' kernels are basically what they should be (better on one vendor's GPUs than another's). Still need to test an RX 5700/XT for AMD, and a GTX 980 for Nvidia before we can do a push.

File: images/makeproject/project/bin/kaktwoos_validator_compare.py [hunk 1]
```diff
@@ -18,8 +18,8 @@
 print("{}".format(file_1))
 print("{}".format(file_2))
 # Strip the files of unneeded information (checkpoints, workunit) by extracting seed
-seeds_1 = re.findall("Found seed: ([\s\S]*?)\n", file_1.stderr_out)
-seeds_2 = re.findall("Found seed: ([\s\S]*?)\n", file_2.stderr_out)
+seeds_1 = re.findall("Found seed: ([\s\S](:?\n|\r\n))\n", file_1.stderr_out)
```
[the remark this answers is not in the document]
Comment (hube12) [reply images/makeproject/project/bin/kaktwoos_validator_compare.py]: oopsi it's (?: not (:?

**Pull request closed without merging**